### DISCUSSION TITLE
How can I clone ring v0.16.20

### DISCUSSION THREAD

**Discussion by cravishe** [briansmith/ring#1930] [General]
Hi
where can I find the source code for ring v0.16.20. Any info on this will be helpful

**Comment by briansmith** [comment]
I recommend using `cargo vendor` or a similar tool. I believe the commit for 0.16.20 has commit message "0.16.20" or similar.

**Reply by cravishe** [reply]
@briansmith , 
sure, thank you for the information. I will try that.
